src/seedu/addressbook/parser/Parser.java

```diff
@@ -15,6 +15,7 @@
 import seedu.addressbook.commands.ClearCommand;
 import seedu.addressbook.commands.Command;
 import seedu.addressbook.commands.DeleteCommand;
+import seedu.addressbook.commands.GroupCommand;
```

Comment:
This import is in wrong order since import statements are sorted alphabetically :)